host/src/main/java/org/area515/resinprinter/server/Main.java

```diff
@@ -59,6 +59,9 @@ public static void main(String[] args) throws Exception {
 		logger.info("=================================================================");
 
 		int port = HostProperties.Instance().getPrinterHostPort();
+		if (System.getProperty("overrideHostPort") != null) {

```

Comment:
Oo, diff comment, I didn't even know you could do that.

Yes, that's a good point. Let me move the override.